src/controller/java/src/chip/devicecontroller/ChipDeviceController.java

```diff
@@ -344,6 +344,82 @@ public void releaseConnectedDevicePointer(long devicePtr) {
     releaseOperationalDevicePointer(devicePtr);
   }
 
+  public enum GroupKeySecurityPolicy {
```

Comment:
could you move this to new file called GroupKeySecurityPolicy?

Reply:
I understand. I will move.